skfda/exploratory/visualization/_magnitude_shape_plot.py

```diff
@@ -207,11 +228,13 @@ def __init__(self, fdatagrid, **kwargs):
         self._outliers = outliers
         self._colormap = plt.cm.get_cmap('seismic')
         self._color = 0.2
-        self._outliercol = 0.8,
+        self._outliercol = 0.8
         self.xlabel = 'MO'
         self.ylabel = 'VO'
         self.title = 'MS-Plot'
```

Comment:
There are nice ideas in this class, that can be used as part of the API in general. For example, having a `title` property, initialized to the dataset name (and not to the type of plot) but that can be modified by the user to change the plot title (or remove it). Same with `xlabel` and `ylabel`. Have you though on this?

Reply:
I was not planning on adding extra features to the current versions as I wanted to arrive to the ordinary handing over date of the project

Reply:
Ok, add it to future work and create an issue if you think that this design is desirable. In this plot, I would change the title to the dataset name.

Reply:
Ok perfect

Reply:
i have solved the issues with the example, i think this file is now okay